documentation/src/main/asciidoc/topics/config_examples/config_inline_jgroups.xml

```diff
@@ -1,8 +1,8 @@
 <infinispan> <1>
   <jgroups> <2>
-    <stack name="prod"> <3>
+    <stack name="prod" extends="tcp"> <3>
```

Comment:
Same thing regarding trimming the rest of the protocols that are duplicate between prod and tcp.

Reply:
I think I also need to modify the asciidoc here, that's why I didn't make more extensive changes.